specification/web/resource-manager/Microsoft.Web/stable/2020-09-01/CommonDefinitions.json

```diff
@@ -0,0 +1,2903 @@
+{
```

Comment:
Why is this file needed?

Reply:
I added it because the ResourceProvider.json file references "./CommonDefinitions.json#/definitions" in many areas

Reply:
@naveedaz although I did realise I forgot to bump the version of CommonDefinitions in the readme... fixed. Please lemme know if I should do otherwise!

Reply:
also @naveedaz do you know what the cause of the Linter Error is after bumping the CommonDefinitions in the readme? I can't seem to figure it out from the error msg